src/components/Registration/DefineRegistrationSummary.vue

```diff
@@ -124,9 +132,14 @@ export default class DefineRegistrationSummary extends Mixins(DateMixin, EnumMix
     return this.getRegistration.defineBusinessValid
   }
 
-  /** The nature of business. */
+  /** The nature of business (code + desc). */
   get natureOfBusiness (): string {
-    return this.getRegistration.naics.naicsDescription
+    const naicsCode = this.getRegistration.naics.naicsCode
+    const naicsDescription = this.getRegistration.naics.naicsDescription
+    if (naicsCode && naicsDescription) {
+      return `${naicsCode} - ${naicsDescription}`
+    }
+    return null
```

Comment:
Returning null here is sort of something I learned along the way: Where do you look for static text (eg, "Not entered")? In the template. Where do you look for variables? In the script. So I have the variable manipulation in the script and the fallback value in the template.